week-4-practise/level-1/exercise-1.js

```diff
@@ -36,9 +36,45 @@ var writers = [
     occupation: "writer",
     age: 64,
     alive: true
-  },
+  }
 ];
+var livingWriters = [];
+
+var writer = "writer";
+
+function isAlive(writer) {
+  return writers

```

Comment:
There is a small difference in spelling between the `writers` variable you're using in line 46 and the `writer` parameter you've defined in line 45. Therefore you're *not* using the value that is passed into the function but instead you're accessing a variable from outside the function scope, called the global scope. Try to avoid accessing any values from the global scope and instead pass all values a function needs in as an argument. Hope this makes sense?